clearpath_control/launch/control.launch.py

```diff
@@ -43,8 +43,10 @@
     ('joint_states', 'platform/joint_states'),
     ('dynamic_joint_states', 'platform/dynamic_joint_states'),
     ('platform_velocity_controller/odom', 'platform/odom'),
+    ('platform_velocity_controller/cmd_vel', 'platform/cmd_vel'),
     ('platform_velocity_controller/cmd_vel_unstamped', 'platform/cmd_vel_unstamped'),
```

Comment:
Should we remove unstamped since we are dropping support for it?

Reply:
I vote yes to this proposal; for Jazzy onwards we should probably drop all unstamped messages. Supporting both was useful during the transition, but I believe the latest ROS control guidelines are for stamped-only.